src/types.ts

```diff
@@ -7,8 +7,17 @@ export type HyphenProviderOptions = {
   environment: string;
   /** The Hyphen server URL */
   horizonServerUrls?: string[];
+  /** The cache options for the provider */
+  cache?: {
+    /** The time-to-live (TTL) in seconds for the cache. */
+    ttlSeconds?: number;
+    /** Generate a cache key function for the evaluation context. */
+    generateCacheKey?: GenerateCacheKeyFn;
```

Comment:
```suggestion
    generateCacheKeyFn?: GenerateCacheKeyFn;
```

this is a nit pick but the existing name makes it seem like a bool value. Not everyone will be using typescript and have the types